### PR TITLE
Create science-and-technology-for-the-built-environment.csl

### PR DESCRIPTION
**Pull request by mitchpaulus**
This is a .csl file for the journal "Science and Technology for the Built Environment" - A Taylor and Francis journal for ASHRAE.

### CODE REVIEW

File: science-and-technology-for-the-built-environment.csl [hunk 1]
```diff
@@ -0,0 +1,549 @@
+<?xml version="1.0" encoding="utf-8"?>
+<style xmlns="http://purl.org/net/xbiblio/csl" class="in-text" version="1.0" demote-non-dropping-particle="never">
+<!-- This style was edited with the Visual CSL Editor (http://editor.citationstyles.org/visualEditor/) -->
+  <info>
+    <title>Science And Technology for the Built Environment</title>
```
Comment (rmzelle) [inline science-and-technology-for-the-built-environment.csl]: @mitchpaulus, small nitpick, but the "And" here should be lowercased.

File: science-and-technology-for-the-built-environment.csl [hunk 2]
```diff
@@ -0,0 +1,549 @@
+<?xml version="1.0" encoding="utf-8"?>
+<style xmlns="http://purl.org/net/xbiblio/csl" class="in-text" version="1.0" demote-non-dropping-particle="never">
```
Comment (rmzelle) [inline science-and-technology-for-the-built-environment.csl]: @mitchpaulus, if this style is only used in a single language, it should have a `default-locale` attribute, see https://github.com/citation-style-language/styles/wiki/Style-Requirements#3---style-locale

File: science-and-technology-for-the-built-environment.csl [hunk 3]
```diff
@@ -0,0 +1,549 @@
...
+    <author>
+      <name>Mitchell T. Paulus</name>
+      <email>paulusm14@gmail.com</email>
+    </author>    
+    <category citation-format="author-date"/>
+    <category field="psychology"/>
+    <category field="generic-base"/>
+    <updated>2016-03-14T14:54:37+00:00</updated>
+    <rights license="http://creativecommons.org/licenses/by-sa/3.0/">This work is licensed under a Creative Commons Attribution-ShareAlike 3.0 License</rights>
+	<issn>2374-4731</issn>
```
Comment (rmzelle) [inline science-and-technology-for-the-built-environment.csl]: https://www.ashrae.org/resources--publications/science-and-technology-for-the-built-environment shows `ISSN: 2374-4731 (Print); 2374-474X (Online)`

Please also include the online ISSN with `<eissn>2374-474X</eissn>`

**Comment by mitchpaulus** [comment]
My apologies for missing items -- Thanks for the help. Issues from the comments should be addressed.

**Comment by rmzelle** [comment]
Great, thanks!

As a final request, could you remove `<category field="generic-base"/>`? We normally only use that for styles that don't belong to any field in particular (such as university-wide styles, or common style guides like Chicago Manual of Style).

**Comment by rmzelle** [comment]
Great, thanks!

**Pull request merged**